modules/ROOT/pages/try-scope-concept.adoc

```diff
@@ -49,6 +49,90 @@ A new transaction is started every time the scope is executed.
+
+[source,xml,linenums]
+----
+<flow name="someFlow">
+	<jms:listener config-ref="JMS_Config" destination="test.in" transactionalAction="ALWAYS_BEGIN"/>
+	<!-- Processors -->
+	<try transactionalAction="?">
+		<!-- Processors -->
+		<!-- Join if possible is the default value for jms:publish operation --> 
+		<jms:publish config-ref="JMS_Config" destination="test.out"/>
+		<raise-error type="APP:SOME"/>
+		<!-- More Processors -->
+	</try>
+	<!-- Processors -->
+	<error-handler>
+		<on-error-continue/>
+	</error-handler>
+</flow>
+----
+
+We analyze this new scenario: 
+* Ignore (`INDIFFERENT`): Once again, the transaction is _suspended_, then message is sent in the `jms:publish` operation regardless of the error caused by the `raise-error` processor. Then, as the error is handled in the `on-error-continue` Error Handler, the transaction is commited and the message is consumed. 
+* Always Begin (`ALWAYS_BEGIN`): Since the transaction that runs within the Try Scope is a different one from the flow, this one is rollbacked (since it is not handled) and the error propagated and handled at flow level. Therefore, the message will still be consumed, but the message sent with the `jms:publish` operation not (that transaction is rollbacked)
+* Begin or Join (`BEGIN_OR_JOIN`): As mentioned in the previous example, the commit of the `jms:publish` message depends on the flow. Since the error is now handled by an `on-error-continue` Error Handler, the transaction will be commited. Thus, both the message read from the `jms:listener` source will consumed and the message sent with `jms:publish` will actually be sent.
+
+Suppose now, instetad of adding an Error Handler at flow level, we add it in the try scope, and the error is thrown 
+after the execution of the it:
+
+[source,xml,linenums]
+----
+<flow name="someFlow">
+	<jms:listener config-ref="JMS_Config" destination="test.in" transactionalAction="ALWAYS_BEGIN"/>
+	<!-- Processors -->
+	<try transactionalAction="?">
+		<!-- Processors -->
+		<!-- Join if possible is the default value for jms:publish operation --> 
+		<jms:publish config-ref="JMS_Config" destination="test.out"/>
+		<!-- More Processors -->
+		<!-- There could be a processor that raises an error, it will be handled by the error handler --> 
+		<error-handler>
+			<on-error-continue/>
+		</error-handler>
+	</try>
+	<!-- Processors -->
+	<raise-error type="APP:SOME"/>
+</flow>
+----
+
+In all this cases the transaction created at Flow level will be rollbacked, and the message will be available to be read again. We analyze what happends with the Try Scope:
+
+* Ignore (`INDIFFERENT`): Once again, the transaction is _suspended_, then message is sent in the `jms:publish` operation regardless of the error caused by the `raise-error` processor. 
+* Always Begin (`ALWAYS_BEGIN`): Since the transaction that runs within the Try Scope is a different one from the flow, this one is commited, and the message sent. 
```

Comment:
this one > Which one? the transaction from the Try Scope or the one in the flow? If this refers to the scope please clarify: 

"Since the transaction that runs within the Try Scope is different from the one that runs in the flow, the transaction from the scope is committed and the message is sent."